vsts/vsts.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
     displayName: Linux
     strategy:
       # Change maxParallel to 1 make builds run in serial rather than in parallel
-      maxParallel: 3
+      maxParallel: 100
```

Comment:
That's a big jump. Are we sure 100 is safe? I don't know much about the impact of changing this.

Reply:
This just means that if we add another framework to our parallel suite, we won't need to increment this max parallel value. It doesn't mean that we will run 100 agents in parallel right now. We'll just have one agent in parallel per supported framework + OS